src/server/pachyderm_test.go

```diff
@@ -2652,6 +2652,10 @@ func testGetLogs(t *testing.T, enableStats bool) {
 
 	c := getPachClient(t)
 	defer require.NoError(t, c.DeleteAll())
+	iter := c.GetLogs("", "", nil, "", false, false, 0)
+	for iter.Next() {
+	}
+	require.NoError(t, iter.Err())
```

Comment:
So it seems like this now tests the case where you just get back the pachd logs, right?

Reply:
Yup